README.md

```diff
@@ -1,36 +1,32 @@
 # Heroku Django Starter Template
 
-An utterly fantastic project starter template for Django 1.11.
+An utterly fantastic project starter template for Django 2.2.
 
 ## Features
 
 - Production-ready configuration for Static Files, Database Settings, Gunicorn, etc.
 - Enhancements to Django's static file serving functionality via WhiteNoise.
-- Latest Python 3.6 runtime environment. 
+- Latest Python 3.7 runtime environment.
 
 ## How to Use
 
-To use this project, follow these steps:
+Try the [Cheat Script](https://github.com/RadialDevGroup/heroku-django-template/wiki/Cheat-Script-(django_new))
 
-1. Create a directory for your project(`$ mkdir projectname;cd projectname`) 
-2. Create your working environment. (`$ python3 -m venv env`)
-3. Activate the environment. (`$ source env/bin/activate`)
-4. Install Django (`$ pip install django`)
-5. Create a new project using this template
+Or, follow these steps:
 
-## Creating Your Project
+1. Create a directory for your project(`$ mkdir projectname;cd projectname`)
+2. Activate the environment. (`$ pipenv shell`)
```

Comment:
Is a step missing?
```
~$mkdir testDjango;cd testDjango
testDjango$pipenv shell
-bash: pipenv: command not found
```

Reply:
kindof... if you haven't installed pipenv before you have to install it first...

Most python projects are moving to pipenv now (I have a section in the journey for it) but it's still pretty new. Maybe I should have a note somewhere about it?

Reply:
I mean I thought I've used it before... but having some sort of prerequisite section in this ReadMe, or a way to check if you have it might be good.